centreon/www/include/eventLogs/export/Request.php

```diff
@@ -442,6 +456,12 @@ public function getSvcMsgStatusSet(): array
         if ($this->getUnknown() === 'true') {
             $this->svcMsgStatusSet[] = sprintf("'%s'", self::STATUS_UNKNOWN);
         }
+        if ($this->getAck() === 'true') {
+            $this->svcMsgStatusSet[] = sprintf("'%s'", self::STATUS_ACKNOWLEDGEMENT);
+        }
+        if ($this->getNotif() === 'true') {
```

Comment:
```suggestion
        if ($this->getNotification() === 'true') {
```

PHP Error: Undefined method getNotif